docs/analytics_onboarding/overview.md

```diff
@@ -32,8 +32,23 @@
 - [ ]  **siuba** - Recommended data analysis library | ([Docs](siuba))
 - [ ]  [**shared_utils**](https://github.com/cal-itp/data-analyses/tree/main/_shared_utils) - A shared utilities library for the analytics team | ([Docs](shared-utils))
 
+**Caltrans Employee Resources:**
+
+- [ ]  [**OnRamp**](https://onramp.dot.ca.gov/) - Caltrans employee intranet
+- [ ]  [**Service Now (SNOW)**](https://cdotprod.service-now.com/sp) - Caltrans IT Service Management Portal for IT issues and requesting specific software
+- [ ]  [**Cal Employee Connect**](https://connect.sco.ca.gov/) - State Controller's Office site for paystubs and tax information
+- [ ]  [**Geospatial Enterprise Engagement Platform - GIS Account Request Form**](https://sv03tmcpo.ct.dot.ca.gov/portal/apps/sites/#/geep/pages/account-request) (optional) - User request form for ArcGIS Online and ArcGIS Portal accounts
+
 &nbsp;
 (get-help)=
-```{admonition} Still need access to a tool above?
+```{admonition} Still need access to a non-Caltrans tool above?
```

Comment:
Can this be updated to just refer to the `#services-team` Slack channel? With the Jarvus contract winding down we're going to phase out references to specific Jarvus team members

Reply:
Yes, I see a few different spots on the docs in addition to this page where that can be updated in another pr!

Reply:
We're also making these kinds of updates right now so we can handle it, I just figured I'd mention since the text box was being touched